app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterMovementReferenceNumberController.scala

```diff
@@ -74,45 +75,33 @@ class EnterMovementReferenceNumberController @Inject() (
       }
     }
 
-  protected def renderMrnPage(answers: MovementReferenceNumberAnswer, isAmend: Boolean)(implicit
+  protected def renderMrnPage(answers: Option[MovementReferenceNumber], isAmend: Boolean)(implicit
```

Comment:
This isn't really a big change because `MovementReferenceNumberAnswer` used to contain an optional type anyway.